runtime/include/vendor_version.h

```diff
@@ -20,7 +20,11 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-/* Example usage for inclusion of a vendor name and repository sha.  These values
+/* This file provides a mean to provide vendor specific version info such as 
```

Comment:
`This file provides a means to set vendor specific...`

Reply:
Yeah, I noticed that as well, will update it.